### DISCUSSION TITLE
Probably not bug but need advice?

### DISCUSSION THREAD

**Discussion by Odan1988** [PRProd/HA-Firemote#197] [Q&A]
### Describe the bug

Can't turn volume up down or mute

### To Reproduce

Try volume up down mute

### Expected Behavior

Volume up down mute

### Which Type of Device Do You Own?

Fire cube 3, jbl bar 1000, FireCube to HDMI on bar then bar to earc. Fire remote works for volume up down mute, firemote does not turn volume up down or mute. I've checked for cec options on the cube and my TV. Shouldn't cec be able to do this through the firemote? 

### Troubleshooting Steps Completed

Did try making a new firemote card and seeing if that would fix it. 

### Additional context

I tried with much older Vizio soundbar I had fire cube could control soundbar firemote could not. Quite possible I'm missing something completely stupid, but also didn't know where to ask, the forum post saw you say you don't check often. Also btw the Syncler Button works perfectly thank you so much, that turn around was quick as hell, super appreciate it.

**Comment by PRProd** [comment]
Is there any chance that your cube doesn't have CEC enabled?  

https://github.com/PRProd/HA-Firemote/wiki/Using-CEC-Controls#firetv-stick

**Reply by PRProd** [reply]
It looks like your soundbar might be capable of getting firmware updates too.  Not sure if that helps...
 https://www.jbl.com/soundbars/BAR-1000-.html

https://www.jbl.com/on/demandware.static/-/Sites-masterCatalog_Harman/default/dwb043f356/pdfs/JBL_HA_Bar%201000_OM_V10_EN.pdf

**Comment by Odan1988** [comment]
![PXL_20230928_125012748](https://github.com/PRProd/HA-Firemote/assets/141099074/501ee645-ca86-47cb-826c-42572e2d1302)

Ok so it is checked correctly. Soundbar is connected to jbl one app and ahtoupdares firmware.

**Reply by Odan1988** [reply]
I've ordered new HDMI cables to be safe as well. They were 8k 60hz 2.1 but bought better brand to see if that's causing the cec issues

**Comment by PRProd** [comment]
It could be the cables, I've seen strange issues resolved by swapping out HDMI cables.

Did you happen to try the other suggestion from the [link](https://github.com/PRProd/HA-Firemote/wiki/Using-CEC-Controls#firetv-stick) too?
![image](https://github.com/PRProd/HA-Firemote/assets/25486599/85d4a9af-cddd-4ad3-ac3a-5a902658788b)
![image](https://github.com/PRProd/HA-Firemote/assets/25486599/26bb01d7-eaff-4859-8093-9aff5f168f64)

**Comment by Odan1988** [comment]
Ok, so good news is it's working. Bad news I might be an idiot. I reset my FireCube and setup equipment control with only the TV as a source did not make the FireCube learn the soundbar. Because tv controls soundbar cube controls tv hence volume and mute work now.

**Reply by PRProd** [reply]
Isn't tech fun sometimes?  I have a friend that says "Computers are dumb.  They only do exactly what you tell them."  It's so true.

Mostly, I'm glad you got it sorted out!  Thanks for asking for my help.

**Reply by Odan1988** [reply]
No thank you for the awesome card man. The one thing in my life I don't lose is my phone lol, I spent thirty minutes looking for the physical remote to test today hahaha. So this working is amazing. Thanks for working with me and the Syncler button is working perfect.
